dbt/models/default/schema/default.vw_pin_value.yml

```diff
@@ -69,12 +69,12 @@ models:
           column_name: certified_class
           config:
             where: CAST(year AS int) < {{ var('test_qc_year_start') }}
-            error_if: ">13" # as of 2024-03-15
+            error_if: ">15" # as of 2024-03-15
```

Comment:
A few of these tests started failing once `test_qc_year_start` got bumped to 2024. My assessment is that the `certified` values represent two new true failures in the underlying data, but the `board` values should actually be tested on the prior year, since board values are finalized much later than mailed/certified values. But correct me if I'm wrong!

Reply:
Yes I suppose that's right: there won't be any 2024 board values until next year, so it doesn't really make sense to test for them.

As an aside, we should change or remove the date on these static thresholds.

Reply:
Oh, the commented date on here for the error threshold is `# as of 2024-03-15` but it wasn't updated with the change. My suggestion was just to update it to `# as of 2024-04-16` or similar.

Reply:
Gotcha, I removed them in 250050ec5bc2b88bff74e98a993a85c864554c78 since we can always inspect the git diff if we want to see when a threshold was set.